docs/pages/guides/includes.mdx

```diff
@@ -12,6 +12,9 @@ export default ({ children }) => <Documentation {...meta}>{children}</Documentat
 JavaScript files can be added to the `includes/` folder to define simple scripts, constants or macros that can be reused across your project.
 Each file in the includes folder will be made available for use within your other SQL or JavaScript files.
 
+It's also possible to add JavaScript to a `.sqlx` file by wrapping it in a `js {...}` block.
```

Comment:
@BenBirt what do you think of adding it here?

Reply:
LGTM, thanks!